process-document/src/main/java/com/ritense/processdocument/service/DocumentDefinitionProcessLinkService.java

```diff
@@ -21,13 +21,14 @@
 import com.ritense.processdocument.domain.impl.request.DocumentDefinitionProcessLinkResponse;
 import com.ritense.processdocument.domain.impl.request.DocumentDefinitionProcessRequest;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface DocumentDefinitionProcessLinkService {
 
-    DocumentDefinitionProcess getDocumentDefinitionProcess(String documentDefinitionName);
+    List<DocumentDefinitionProcess> getDocumentDefinitionProcess(String documentDefinitionName);
```

Comment:
This is a breaking change. Is it possible to create this method next to the existing one?

Reply:
Changed